src/styles/index.scss

```diff
@@ -1,3 +1,125 @@
-body {
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+body,
+h2,
+p {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  width: 200px;
+  margin: auto;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  justify-content: center;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    text-align: left;
+    padding-top: 40px;
+    color: #060b35;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    padding-top: 4px;
+  }
+
+  &__stars-rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-top: 16px;
+    line-height: 18px;
+  }
+
+  &__cost {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-top: 24px;
+    line-height: 18px;
+    padding-bottom: 16px;
+  }
+
+  &__cost-title {
+    font-weight: 400;
+    text-align: left;
+  }
+
+  &__cost-price {
+    font-size: 16px;
+    font-weight: 700;
+    text-align: right;
+  }
+
+  &__buttom {
+    box-sizing: border-box;
+    display: flex;
+    background-color: #00acdc;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    color: #fff;
+    text-decoration: none;
+    text-transform: uppercase;
+    justify-content: center;
+    align-items: center;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+
+    &__:hover {
+      border-color: #00acdc;
+      color: #f3f3f3;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(/images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    align-items: left;
+    margin-right: 4px;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &--4 .stars__star:nth-child(-n+4) {
```

Comment:
Implement this requirement from task. 
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/9cbfdeed-a959-4b4f-b393-35ba466332d3)
now it doesn't work 
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/0aa37eb7-c6f1-49ed-ac8b-156b96bc2931)